### ISSUE TITLE
Failed to load the autocomplete-bibtex package

### ISSUE THREAD

**Issue by kumazawajiro** [apcshields/autocomplete-bibtex#79]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Ubuntu 16.04.2
**Thrown From**: [autocomplete-bibtex](https://github.com/apcshields/autocomplete-bibtex) package 0.7.0

### Stack Trace

Failed to load the autocomplete-bibtex package

```
At Cannot find module 'zotero-bibtex-parse'

Error: Cannot find module 'zotero-bibtex-parse'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/opt/atom/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/opt/atom/resources/app/src/module-cache.js:354:58)
    at Module.require (opt/atom/resources/app/static/index.js:40:43)
    at require (/opt/atom/resources/app/src/native-compile-cache.js:66:33)
    at /packages/autocomplete-bibtex/lib/provider.coffee:2:15)
    at /packages/autocomplete-bibtex/lib/provider.coffee:1:1)
    at Module.get_Module._compile (/opt/atom/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .coffee] (/opt/atom/resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (opt/atom/resources/app/static/index.js:47:45)
    at require (/opt/atom/resources/app/src/native-compile-cache.js:66:33)
    at /packages/autocomplete-bibtex/lib/main.coffee:4:21)
    at /packages/autocomplete-bibtex/lib/main.coffee:1:1)
    at Module.get_Module._compile (/opt/atom/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .coffee] (/opt/atom/resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (opt/atom/resources/app/static/index.js:47:45)
    at require (internal/module.js:20:19)
    at customRequire (/opt/atom/resources/app/static/<embedded>:96:26)
    at Package.module.exports.Package.requireMainModule (/opt/atom/resources/app/src/package.js:887:35)
    at /opt/atom/resources/app/src/package.js:158:34
    at Package.module.exports.Package.measure (/opt/atom/resources/app/src/package.js:99:21)
    at Package.module.exports.Package.load (/opt/atom/resources/app/src/package.js:144:18)
    at PackageManager.module.exports.PackageManager.loadAvailablePackage (/opt/atom/resources/app/src/package-manager.js:565:22)
    at /opt/atom/resources/app/src/package-manager.js:479:25
```

### Commands

```
```

### Non-Core Packages

```
atom-beautify 0.30.0 
atom-clock 0.1.9 
atom-html-preview 0.1.22 
atom-pronama-chan 1.0.1 
atomic-chrome 0.3.0 
auto-encoding 0.7.0 
autoclose-html 0.23.0 
autocomplete-bibtex 0.7.0 
autocomplete-clang 0.11.3 
autocomplete-haskell 0.7.2 
autocomplete-paths 1.0.5 
busy-signal 1.4.3 
color-picker 2.2.5 
docblockr 0.10.3 
editor-stats 0.17.0 
editorconfig 2.2.2 
export-html 3.1.0 
file-icons 2.1.7 
git-history 3.3.0 
haskell-ghc-mod 1.19.9 
hey-pane 1.0.0 
highlight-line 0.12.0 
highlight-selected 0.13.1 
Hydrogen 1.17.1 
ide-haskell 1.9.6 
intentions 1.1.2 
language-gnuplot-atom 0.2.0 
language-haskell 1.12.1 
language-latex 1.0.0 
language-restructuredtext 1.1.0 
language-source-cfg 0.3.1 
latex 0.44.0 
latexer 0.3.0 
latextools 0.8.5 
linter 2.2.0 
linter-clang 4.1.1 
linter-csslint 1.3.4 
linter-hlint 1.0.1 
linter-htmlhint 1.3.3 
linter-pycodestyle 2.0.2 
linter-ui-default 1.6.1 
linter-write-good 0.9.0 
local-history 4.3.0 
minimap 4.28.2 
minimap-find-and-replace 4.5.2 
minimap-highlight-selected 4.6.1 
minimap-pigments 0.2.2 
open-recent 5.0.0 
pdf-view 0.59.0 
pigments 0.39.1 
project-manager 3.3.5 
quick-highlight 0.10.0 
regex-railroad-diagram 0.17.0 
rst-preview-pandoc 0.1.12 
save-session 0.15.7 
script 3.15.0 
scroll-through-time 0.2.0 
show-ideographic-space 1.0.2 
sort-lines 0.14.0 
split-diff 1.4.1 
symbols-tree-view 0.14.0 
tablr 1.8.3 
term3 0.22.1 
toggler 0.3.0 
trailing-spaces 0.4.0 
web-search 0.3.3 
web-view 3.0.0 
wordcount 2.10.4 
```

**Comment by vitasiku** [comment]
I followed the following steps to get it working for me.

1. clone autocomplete-bibtex
2. cd autocomplete-bibtex
3. npm install zotero-bibtex-parse 
4. From inside autocomplete-bibtex, edit package.json and
change to  zotero-bibtex-parse": "0.0.13", 
I know 0.0.13 is an old version but unfortunately it's the only one I managed to install via npm.
5. apm install 
6. apm link
7. Restart Atom

Hope that helps.
